doc/news/_preparation_next_release.md

```diff
@@ -71,10 +71,20 @@ Thanks to Armin Wurzinger.
 
 ## Plugins
 
+### CCode
+
+- We fixed various warnings in the source code reported by [OCLint](http://oclint.org). *(René Schwaiger)*
+- The plugin now also encodes and decodes key names in addition to key values. *(René Schwaiger)*
+
 ### Crypto
 
 - The `crypto` plugin now uses Elektra's `libinvoke` and the `base64` plugin in order to encode and decode Base64 strings. This improvement reduces code duplication between the two plugins. *(Peter Nirschl)*
 
+### Haskell
+
+- An issue when building Haskell plugins with a cached sandbox is fixed in case
```

Comment:
Thank you for reordering this. We should discuss it in the next meeting how to use your new organization.